packages/components/scss/BaseStyleSheet.scss

```diff
@@ -16,6 +21,14 @@ html {
   min-height: 100%;
 }
 
+// override
+.text-black-50 {
+  color: background-opacity(50) !important;
```

Comment:
Wonder if this should actually use `dh-color-black` and `dh-color-white` instead... unsure where these are used, you probably know better than I.